Makefile

```diff
@@ -13,6 +13,13 @@ imports = \
 xcodeproj:
 	swift package generate-xcodeproj
 
+bootstrap: common-crypto-mm xcodeproj
+
+common-crypto-mm:
+	-@mkdir "$(COMMON_CRYPTO_PATH)" || true
+	-@touch "$(COMMON_CRYPTO_MODULE_PATH)" || true
+	-@echo "$$COMMON_CRYPTO_MODULE_MAP" > "$(COMMON_CRYPTO_MODULE_PATH)"
```

Comment:
so this creates the `CommonCrypto.framework` directory, then adds a module map file, and then puts the module map code into the file.